php.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
I got `syntax error, unexpected '<', expecting end of file` in PHP 7.0.8, does it need an ending `?>`?

Reply:
https://repl.it/repls/VillainousJadedExpertise

Reply:
Because of the type hinting it requires PHP 7.1+, as you can see here: https://3v4l.org/ceMYL. A 5.6 compatible version looks like this: https://3v4l.org/2UUie

Reply:
Ah, I see, 7.1+ usage is fine. Can you add a comment at the beginning with the PHP version for clarity?

Reply:
Comment added!